### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by yanksyoon**
Applicable spec: <link>

### Overview

Updates README.md to conform to latest standards.

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [x] The documentation for charmhub is updated.
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,63 @@
...
+
+After having deployed and integrated the charm with the GitHub runner charm, the SSH connection to
+the runner VM should be accessible via [action-tmate](https://github.com/canonical/action-tmate).
+To find out more about how to configure the workflow, refer to the [Getting Started](https://github.com/canonical/action-tmate?tab=readme-ov-file#getting-started) section of action tmate.
+
+## Integrations
+<!-- Information about particularly relevant interfaces, endpoints or libraries related to the charm. For example, peer relation endpoints required by other charms for integration.--> 
+* debug-ssh: Provides GitHub runners with addresses to connect to for action-tmate.
+
+For a full list, please refer to the full list on [CharmHub](https://charmhub.io/tmate-ssh-server/integrations).
```
Comment (erinecon) [inline README.md:53]: I would rephrase slightly:
`For a full list of integrations, please refer to the [Charmhub documentation](https://charmhub.io/tmate-ssh-server/integrations).`

**Review by erinecon — COMMENTED** [review]
Thanks for updating the README! It looks great, just one comment from me :)

**Review by erinecon — DISMISSED** [review]
Thanks for updating the README!

File: README.md [hunk 2]
```diff
@@ -1,45 +1,63 @@
...
+## Integrations
+<!-- Information about particularly relevant interfaces, endpoints or libraries related to the charm. For example, peer relation endpoints required by other charms for integration.--> 
+* debug-ssh: Provides GitHub runners with addresses to connect to for action-tmate.
+
+For a full list of integrations, please refer to the [Charmhub documentation](https://charmhub.io/github-runner-image-builder/integrations).
+
+## Learn more
+* [Read more](https://charmhub.io/tmate-ssh-server) <!--Link to the charm's official documentation-->
+* [Developer documentation](https://github.com/canonical/tmate-ssh-server-operator/blob/main/CONTRIBUTING.md) <!--Link to any developer documentation-->
+* [Troubleshooting](https://matrix.to/#/!DYvOMMMjuXPZRJYHdy:ubuntu.com?via=ubuntu.com&via=matrix.org)
```
Comment (amandahla) [inline README.md:58]: Same here, troubleshooting links to the matrix room.

Comment (erinecon) [reply README.md:58]: This is expected; this is what I've been telling people to link to if there's no official troubleshooting documentation/guide.

Comment (varshigupta12) [reply README.md:58]: @erinecon This links to our private team room in Matrix. Would be better to link to the Charm Development channel on Matrix_
https://matrix.to/#/#charmhub-charmdev:ubuntu.com

Comment (erinecon) [reply README.md:58]: Ahhhh okay. That's a good catch @varshigupta12 and @amandahla , thanks!

**Review by erinecon — APPROVED** [review]

**Review by amandahla — APPROVED** [review]

**Comment by Thanhphan1147** [comment]
Hi @yanksyoon, it seems like there's a lint issue, can you take a look? Thanks!
